styles.css

```diff
@@ -0,0 +1,287 @@
+.background-theme{
+    height:100% ;
+    width: 100%;
+    position: absolute;
+    
+}
+.background-top{
+    height: 300px;
+    background-color: rgb(120, 167, 120);
+}
+.background-bottom{
+    height: 2000px;
+    background-color: rgb(216, 215, 215);
+   
+}
+body{
+    margin: 0;
+    padding: 0;
+}
+.main-container{
+    width: 1100px;
+    margin: 0 auto;
+    position: relative;
+}
+.nav-bar{
+    height: 300px;
+    width: 70%;
+    margin-left: 15%;
+    margin-right: 15%;
+}
+.grid-left{
+    padding-top: 20px;
+    color: white;
+    width: 50%;
+}
+.grid-right{
+    float: right;
+    margin-top: -60px;
+}
+ul{
+    list-style: none;
+}
+.grid-right li{
+    display: inline;
+}
+.profile-container{
+    background-color: white;
+    height: 315px;
+    width: 70%;
+    margin-left: 15%;
+    margin-right: 15%;
+    margin-top: -120px;
+}
+.profile-container .column {
```

Comment:
Don't prefer to use this kind of class definition, use it only when you have other tags with the same class names
try making class names as unique as possible,